openapi-generator-annotations/lib/src/openapi_generator_annotations_base.dart

```diff
@@ -25,6 +31,19 @@ class Openapi {
   /// -i
   final String inputSpecFile;
```

Comment:
With InputSpec, we can now deprecate this. It will be removed in version 5.0.0 (which will be released next week)